.openpublishing.redirection.json

```
@@ -6929,6 +6929,11 @@
       "source_path": "exchange/virtual-folder/exchange/Set-DefaultTenantBriefingConfig.md",
       "redirect_url": "/viva/insights/personal/reference/briefing-pause",
       "redirect_document_id": false
+    },
+    {
+      "source_path": "teams/teams-ps/teams/New-CsTeamsShiftsConnectionTeamMap.md",
```

```suggestion
      "source_path": "teams/virtual-folder/teams/New-CsTeamsShiftsConnectionTeamMap.md",
```
Setting "source_path" to its previous state as my last commit did not resolve the warning status in the PR